src/test/java/com/spotify/docker/client/DefaultDockerClientTest.java

```diff
@@ -4870,6 +4870,28 @@ public void testCreateService() throws Exception {
   }
 
   @Test
+  public void testCreateServiceWithWarnings() throws Exception {
```

Comment:
@davidxia I can add a unit test. It might still be useful to keep the current test. Let me explain: I noticed testing the changes that the docker documentation is incorrect (it states that the field in "warning" and a string instead of "warnings" as an array). A unit test would not have caught this. This might not be the intent of the docker-client tests and the developer should be responsible to test the right contract but let me know if you think that the current test as any value.